### PR TITLE
Updates after moving Auditing Code to Healthcare Shared Components

### PR DESCRIPTION
**Pull request by v-shaaal**
## Description
Initial pass after moving generic auditing code to healthcare shared components.

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Audit/AuditEventTypeMappingTests.cs [hunk 1]
```diff
@@ -11,8 +11,8 @@
 using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.AspNetCore.Mvc.Infrastructure;
 using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Health.Api.Features.Audit;
```
Comment (jackliums) [inline src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Audit/AuditEventTypeMappingTests.cs:14]: These tests should move together with the code right?

Comment (v-shaaal) [reply src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Audit/AuditEventTypeMappingTests.cs:14]: Yep, I will move them. Quick question, should we also keep them here in FHIR or just having them in the shared components would suffice?

Comment (jackliums) [reply src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Audit/AuditEventTypeMappingTests.cs:14]: We only need these in the shared. FHIR has its own integration tests that validates the functionality already so we don't need these unit tests here.

Comment (v-shaaal) [reply src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Audit/AuditEventTypeMappingTests.cs:14]: Makes sense. I will delete these tests from here. I have already pushed a new iteration to the other pull request (in healthcare shared components) with these tests added.

**Pull request merged**